iModelCore/ECDb/Tests/NonPublished/SchemaRemapTests.cpp

```diff
@@ -6724,4 +6724,521 @@ TEST_F(SchemaRemapTestFixture, InjectBaseClass4_Simplified)
     ASSERT_EQ(JsonValue(R"json([{"ifcCompositionType":"A"}])json"), result);
     }
     }
+
+//---------------------------------------------------------------------------------------
+// @bsimethod
+//+---------------+---------------+---------------+---------------+---------------+------
+TEST_F(SchemaRemapTestFixture, RevitStoryScenario)

```

Comment:
These tests are too large to exist without a description of the test case they are covering.  Either text or a diagram.  

Also it would be ideal if the differences between the new and old schemas was highlighted in the xml or the changes were inserted in the xml with some print f string.  As it is you have to copy them out and diff to see the difference.  Another option would be to save them as test data external to the test.  None of these options are great but we have to do better for our future selves that will have to read and understand them.

Reply:
I've used printf and added an explanation to the schemas.